src/misc_commands.p8

```diff
@@ -88,6 +89,74 @@ misc_commands {
         return true
     }
 
+        sub cmd_highlight_color() -> uword {
+        if main.command_arguments_size==0
+            return err.set("Missing args: strongcolor,promptcolor,errcolor")
+
+        ubyte strcol = conv.str2ubyte(main.command_arguments_ptr) & 15
+        main.command_arguments_ptr += cx16.r15 + 1
+        
+        if strcol==main.COLOR_BACKGROUND
+            return err.set("Highlight and bg color are the same")
+
+        ubyte prptcol = conv.str2ubyte(main.command_arguments_ptr) & 15
+        main.command_arguments_ptr += cx16.r15 + 1
+
+        ubyte errcol = conv.str2ubyte(main.command_arguments_ptr)
+
+        if errcol==main.COLOR_BACKGROUND
+            return err.set("Error and bg color are the same")
+
+        if errcol==main.COLOR_NORMAL
+            return err.set("Error and text color are the same")
+
+        txt.print("Highlight: ")
```

Comment:
I find the output too verbose for such a simple command. It's kinda bloaty for the shell.prg. Why not just print nothing when colors are changed succesfully? Just like the existing "color" command does